pkg/controller/installation/core_controller.go

```diff
@@ -1359,6 +1359,7 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		Terminating:             nodeTerminating,
 		PrometheusServerTLS:     nodePrometheusTLS,
 		FelixHealthPort:         *felixConfiguration.Spec.HealthPort,
+		NodeCgroupV2Path:        felixConfiguration.Spec.CgroupV2Path,
```

Comment:
I don't see `NodeCgroupV2Path` in the NodeConfiguration, but I do see `NodeBPFFsMountDir`?